migration/mappings-data/account-map.csv.20220722.verified

```diff
@@ -169,3 +169,4 @@ mharwood,markharwood,Mark Harwood
 hossman,hossman,Chris M. Hostetter
 munendrasn,munendrasn,Munendra S N
 vajda,ovalhub,Andi Vajda
+manish1982,manishbafna,Manish
```

Comment:
We don't see the username `manish1982` in Lucene Jira, I wonder if you mean `manish82`?
```
manish82,Manish
```

Reply:
```suggestion
manish82,manishbafna,Manish
```
I suggest this change - @manishbafna if you are fine with it I'll commit this in.

Reply:
<img width="296" alt="image" src="https://user-images.githubusercontent.com/1758199/181168231-9e0d1ac2-b47b-4879-ad94-19db05900bec.png">
My username in JIRA is manish1982.

Reply:
Ok, but`manish1982` has no activity in Lucene Jira so far.
https://issues.apache.org/jira/secure/ViewProfile.jspa?name=manish1982

On the other hand, `manish82` has activities in Lucene Jira.
https://issues.apache.org/jira/secure/ViewProfile.jspa?name=manish82

Two Jira users' names are the same - I suspect you have two Jira accounts?

Reply:
You are right. We can add manish82 to this.

Reply:
Thanks @manishbafna for confirming. I'll commit this in main soon.